app/controllers/v1/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class V1::UsersController < APIController
+  def index
+    user = User.where(auth_id: params.dig(:filter, :auth_id))
```

Comment:
I just looked at what happens when you don't pass a parameter to this, and it does:

`SELECT "users".* FROM "users" WHERE "users"."auth_id" IS NULL`

which is probably not what we want. Can you change this to only filter by `auth_id` if one exists?

Reply:
Thanks @leeky makes sense. I've added the where(nil) and a test for the end point with no parameters.